.github/workflows/deployment.yml

```diff
@@ -115,12 +119,55 @@ jobs:
       with:
         name: JabRef-${{ matrix.displayName }}
         path: build/distribution
-    - name: Upload to builds.jabref.org
-      uses: garygrossgarten/github-action-scp@release
-      with:
-        local: build/distribution
-        remote: www/${{ steps.gitversion.outputs.branchName }}
-        host: builds.jabref.org
-        username: builds_jabref_org
-        privateKey: ${{ secrets.buildJabRefPrivateKey }}
-        port: 9922
+  deploy:
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2-beta
+        with:
+          fetch-depth: 0
+      - name: Fetch tags and master for GitVersion
+        run: |
+          git fetch --tags origin
+          git rev-parse --verify master
+          if (-not $?) {
+            git branch --force --create-reflog master origin/master
+          }
+        shell: pwsh
+      - name: Install GitVersion
+        uses: gittools/actions/setup-gitversion@v0.3
+        with:
+            versionSpec: '5.1.2'
+      - name: Run GitVersion
+        id: gitversion
+        uses: gittools/actions/execute-gitversion@v0.3
+      - name: Get linux binaries
+        uses: actions/download-artifact@master
+        with:
+          name: JabRef-linux
+          path: build/distribution
+      - name: Get windows binaries
+        uses: actions/download-artifact@master
+        with:
+          name: JabRef-windows
+          path: build/distribution
+      - name: Get macOS binaries
+        uses: actions/download-artifact@master
+        with:
+          name: JabRef-macOS
+          path: build/distribution/
+      - name: Deploy to builds.jabref.org
+        id: deploy
+        uses: Pendect/action-rsyncer@v1.1.0
+        env:
+          DEPLOY_KEY: ${{ secrets.buildJabRefPrivateKey }}
+          BRANCH: ${{ steps.gitversion.outputs.branchName }}
+        with:
+          flags: -vaz --itemize-changes --progress --stats --partial-dir=/tmp/partial --rsync-path="mkdir -p /var/www/builds.jabref.org/www/${{ steps.gitversion.outputs.branchName }} && rsync"
```

Comment:
Is `progress` really required? Normally you don't sit in front of the action output and follow the progree